### PR TITLE
Add logs to SQL Rest Handler for better debugging.

### PR DESCRIPTION
**Pull request by jinsoor-amzn**
*Description of changes:*
Following try catch will cause NPE without any logs. This commit helps to better debug NPE issue until we add more meaningful exception back to customer.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/main/java/com/amazon/opendistroforelasticsearch/sql/plugin/RestSqlAction.java [hunk 1]
```diff
@@ -33,6 +35,7 @@
 import java.util.Set;
 
 public class RestSqlAction extends BaseRestHandler {
+    public static Logger logger = LogManager.getLogger(RestSqlAction.class);
```
Comment (qreshi) [inline src/main/java/com/amazon/opendistroforelasticsearch/sql/plugin/RestSqlAction.java]: Might be better to declare this Logger ```private static final```

Comment (jinsoor-amzn) [reply src/main/java/com/amazon/opendistroforelasticsearch/sql/plugin/RestSqlAction.java]: Oops. Good catch. Updated.

Comment (dai-chen) [reply src/main/java/com/amazon/opendistroforelasticsearch/sql/plugin/RestSqlAction.java]: Thanks for the change! Mohammad was referring to the logger convention like this: https://github.com/opendistro-for-elasticsearch/sql/blob/c126e456538167283df92f49850aa93e0307e69a/src/main/java/com/amazon/opendistroforelasticsearch/sql/executor/AsyncRestExecutor.java#L44

I will try to fix the NPE issue this week.

**Comment by jinsoor-amzn** [comment]
Please squash and merge when approved.

File: src/main/java/com/amazon/opendistroforelasticsearch/sql/plugin/RestSqlAction.java (hunk 1 above)
Comment (jinsoor-amzn) [reply src/main/java/com/amazon/opendistroforelasticsearch/sql/plugin/RestSqlAction.java]: I have updated with package convention in third commit.

**Pull request merged**